src/tactic/ui/panel/table_layout_wdg.py

```diff
@@ -2807,6 +2818,8 @@ def handle_select(my, table, sobject):
         td.add_class( 'SPT_DTS' )
         #td.add_color("background-color", "background", -0)
         td.add_color("opacity", "0.5")
+        if my.subscribed_search_keys.get(sobject.get_search_key()):
+            td.add_border(direction="right", color="#ecbf7f", size="2px")
```

Comment:
should we use a subscribe attribute called "subscribed" here instead? can put it in the palette class